jf_agent/git/bitbucket_server.py

```diff
@@ -447,3 +461,5 @@ def get_pull_requests(
                 }
 
                 yield normalized_pr
+        if verbose:
+            print(f"Ending download of PRs for repo {repo}")
```

Comment:
I'm not sure how useful start/end logging is here. I think tqdm should be printing this already 🤔

Reply:
👍